dateparser/languages/locale.py

```diff
@@ -145,6 +170,7 @@ def translate(self, date_string, keep_formatting=False, settings=None):
         if "in" in date_string_tokens:
             date_string_tokens = self._clear_future_words(date_string_tokens)
 
+        self.remove_multiple_occurrences(date_string_tokens)
```

Comment:
What does this do? None of the new test scenarios seem to feature multiple occurrences of a week day. What scenarios does this address? (I wonder if it could be introducing other issues)